project.clj

```diff
@@ -1,10 +1,10 @@
 (def clj-version "1.10.1")
-(def ks-version "3.1.1")
-(def tk-version "3.1.0")
+(def ks-version "3.1.2")
+(def tk-version "3.1.1")
 (def tk-jetty-version "4.1.3")
```

Comment:
Think this also needs to be updated but I don't see a new release out yet w/ the commons-compress bump.

Reply:
We shouldn't need to pull in a version with that, the relevant dependencies are all overridden in the consuming projects.